moveit_core/background_processing/src/background_processing.cpp

```diff
@@ -35,14 +35,14 @@
 /* Author: Ioan Sucan */
 
 #include <moveit/background_processing/background_processing.h>
-#include "rclcpp/rclcpp.hpp"
+#include <rclcpp/logging.hpp>
 
 namespace moveit
 {
 namespace tools
 {
 // Logger
-static const rclcpp::Logger LOGGER = rclcpp::get_logger("moveit_background_processing.background_processing");
```

Comment:
We should use a systematic naming convention for this (see https://github.com/ros-planning/moveit2/blob/master/doc/MIGRATION_GUIDE.md). Right now we decided to use the library name only (here `moveit_background_processing`) without a separator if the file name is identical. I agree that having a separator would make sense for scoping logger levels. Also, I would argue for including the package name, i.e. `moveit.core.background_processing` following the pattern `moveit.<package>.<library>.<file>` where we drop `moveit` from `<package>.<library>.<file>` and remove redundancies in the namespace if there are any. What do you think of this approach?

Reply:
Adding the package name is a good idea. My main aim was to introduce the top-level `moveit` scope.